ui/ui-frontend/projects/vitamui-library/src/lib/models/zip/zip-file.class.ts

```diff
@@ -52,9 +54,14 @@ export class ZipFile {
     };
   }
 
+  setTransactionId(transactionId: string) {
+    this.zipFileStatus.transactionId = transactionId;
+    return this;
+  }
+
   addFiles(files: FileList | File[]) {
```

Comment:
```suggestion
  addFiles(files: FileList | File[]): ZipFile {
```

Reply:
fait, on pourrait meme mettre : this pour etre encore plus precis